### ISSUE TITLE
`randomize()` from `random` not working on JS

### ISSUE THREAD

**Issue by dawkot** [nim-lang/Nim#12418]
### Example
```nim
import random

randomize()
```

### Current Output
```
Uncaught Error: Error: unhandled exception: value out of range [RangeError]
Traceback (most recent call last)
random.randomize, line: 646
random.randomize, line: 593
random.initRand, line: 573
```

### Additional Information
Tested with both 1.0 stable and 1.0.99

**Comment by bluenote10** [comment]
I was already wondering if [the fix](https://github.com/nim-lang/Nim/commit/3bb6d1916e3ebae9196d9dc87787de3e26e9f908) for https://github.com/nim-lang/Nim/issues/11772 would work for all time ;).

**Comment by narimiran** [comment]
> I was already wondering if the fix for #11772 would work for all time ;).

Ha! We're losing the battle against time :) I'm guessing we need to start dividing instead of multiplying....
